docs/components/SideNav.tsx

```diff
@@ -9,14 +9,20 @@ export const SideNav: React.FC<{ className?: string }> = ({ className }) => {
   return (
     <nav className={`sidenav text-sm px-4 w-[240px] ${className ?? ""}`}>
       <div
-        className="sticky pt-8 pb-4 overflow-y-auto"
+        className="sticky pt-8 pb-4 overflow-y-auto sidebar-container"
         style={{
-          height: "calc(100vh - var(--top-nav-height) - 4px)",
           top: "calc(var(--top-nav-height) + 4px)",
         }}
       >
         <SidebarContent />
       </div>
+      <style jsx> {`
+        .sidebar-container {
```

Comment:
Can you please add a comment as to why the `height` is there twice. For future programmers who might want to clean it up.